packages/containers/psammead-timestamp-container/src/timestampUtilities.test.js

```diff
@@ -11,6 +12,14 @@ const timestamp = 1539969006000; // 19 October 2018
 const locale = 'en-gb';
 
 describe('Timestamp utility functions', () => {
+  describe('formatDuration', () => {
+    it('should return duration in expected format', () => {
```

Comment:
Could you also add a test for when you pass a format in? Other than that looks good to me :)

Reply:
Added